examples/gno.land/r/demo/banktest/z_0_filetest.gno

```diff
@@ -44,4 +45,3 @@ func main() {
 //  * g17rgsdnfxzza0sdfsdma37sdwxagsz378833ca4 100000000ugnot sent, 100000000ugnot returned, at 2009-02-13 11:31pm UTC
 //
 // ## total deposits
-// 300000000ugnot
```

Comment:
Why did this change?

Reply:
`z_0_filetest.gno` really doesn't have any logic for `banktest` realm to have that amount of ugnot. Could you examine it?